### ISSUE TITLE
App not creating appdata path.

### ISSUE THREAD

**Issue by Tryflle** [BrianLima/UWPHook#106]
```
Error: Program failed exporting your games:

Could not find a part of the path 'C:\Users\<user>\AppData\Local\Temp\UWPHook\tmp_grid'.   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)

   at System.IO.FileSystemEnumerableIterator`1.CommonInit()

   at System.IO.FileSystemEnumerableIterator`1..ctor(String path, String originalUserPath, String searchPattern, SearchOption searchOption, SearchResultHandler`1 resultHandler, Boolean checkHost)

   at System.IO.Directory.GetFiles(String path)

   at UWPHook.GamesWindow.<>c__DisplayClass12_1.<ExportGames>b__2()

   at System.Threading.Tasks.Task.InnerInvoke()

   at System.Threading.Tasks.Task.Execute()

--- End of stack trace from previous location where exception was thrown ---

   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)

   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)

   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)

   at UWPHook.GamesWindow.<ExportGames>d__12.MoveNext()
```

I glanced through the open issues and didn't see it, sorry if this is a repeat.
Just filing this as a report for you, the solution was to manually create the directory given (%appdatalocal\temp\uwphook\tmp_grid)

**Comment by stevealexandre** [comment]
Use this release : https://github.com/stevealexandre/UWPHook/releases/tag/2.15.1

**Comment by BrianLima** [comment]
Should be fixed as of v2.14.0.
